client/src/pages/profile/SocialsInput/index.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { ReactElement, FC } from 'react';
+import { Grid, Icon, Input } from 'semantic-ui-react';
+
+import './styles.css';
+import { ISocialsInput } from './types';
+
+const SocialsInput: FC<ISocialsInput> = ({
+  icon,
+  value,
+  readOnly,
+  onChange,
+  viewable,
+}): ReactElement => (
+  <>
+    {viewable && (
+      <Grid className="social-input" columns="equal">
+        <Grid.Column className="col" width={2}>
+          <Icon size="big" name={icon} />
+        </Grid.Column>
+        <Grid.Column className="col">
+          <Input
+            size="big"
+            className="input"
+            fluid
+            value={value}
+            readOnly={readOnly}
```

Comment:
I don't think readOnly does anything. I think the prop is called disabled.

Reply:
readOnly is a different prop that disabled. The difference between the two is that disabled completely turns off the input. No interaction with the page. ReadOnly allows you to tab through it and everything but the user can't modify the contents of it. It also preserves the CSS and doesn't darken it